tsconfig.json

```
@@ -0,0 +1,71 @@
+{
+  "include": [
+    "src"
+  ],
+  "compilerOptions": {
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Node 10 has [full ES2017 support](https://node.green/#ES2017). We should also bump the `engines` version in package.json to `node >= 10`, as 8 is out of LTS.

```suggestion
    "target": "ES2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```